src/kOS/Suffixed/VesselTarget.cs

```diff
@@ -464,6 +465,13 @@ private void InitializeSuffixes()
             AddSuffix("MESSAGES", new NoArgsSuffix<MessageQueueStructure>(() => GetMessages()));
         }
 
+        public global::Part GetControlPart()
+        {
+            global::Part res = Vessel.GetReferenceTransformPart(); //this can actually be null if "control from here" has never been used
```

Comment:
Oh, I also didn't try it on a vessel with no core or module, since we have historically been able to control those vessels anyways (purely as a bug) but I could see that being another point where it comes up.  I'm glad it came up in your testing, it might have been hard to track down once it was released.